test/integration/integration_test.rb

```diff
@@ -83,6 +85,7 @@ def ipv6
 
   def purge_cache name
     %x{#{ENV['PDNS_CONTROL'] || "pdns_control"} purge "#{name}"}
-    $? == 0
+    sleep 60 unless $? == 0
```

Comment:
The reasoning behind this is that the powerdns packet cache is a minute by default. This means that if purging fails then we do our best to ensure the cache is clean. Given these are integration tests I think this is acceptable.